core/commands/root.go

```diff
@@ -191,6 +191,7 @@ var rootROSubcommands = map[string]*cmds.Command{
 		Subcommands: map[string]*cmds.Command{
 			"get":     dag.DagGetCmd,
 			"resolve": dag.DagResolveCmd,
+			"stat":    dag.DagStatCmd,
```

Comment:
@Stebalien I notice that `ipfs dag export` isn't listed as a ReadOnly command (https://github.com/ipfs/go-ipfs/pull/7036#discussion_r405716667). This should be no more expensive then a Get so I'm assuming it's fine to expose this. Seem reasonable?

Reply:
@aschmahmann was requested here: https://github.com/ipfs/go-ipfs/pull/7036#discussion_r405692836